google/ggrc-core
# QUICK-FIX Clean up permissions

# Issue description

Dead code and unused configs.

# Steps to test the changes

Nothing should change.

Check that permissions work for the following cases:
1. Admin dashboard opens for Admins and only for Admins.
2. For Creator, object pages open for objects visible and don't open for objects invisible.
3. For Reader, All Objects page shows the same stuff as before.

# Solution description

Unused stuff is removed.

# Sanity checklist




# PR Review checklist
